### ISSUE TITLE
Seems not to work?

### ISSUE THREAD

**Issue by marbetschar** [sochalewski/UIImageViewAlignedSwift#14]
I've tried to integrate your `UIImageViewAlignedSwift` class as follows, but unfortunately the UIImage itself is not rendered at all (see attached screenshot). What's wrong?

```
let imageView = UIImageViewAligned(image: UIImage(named:"my-image"))
imageView.contentMode = .scaleAspectFit
imageView.scaling = .down
imageView.alignment = .bottomRight
imageView.backgroundColor = .red

imageView.translatesAutoresizingMaskIntoConstraints = false

self.view.addSubview(self.rightImageView)

let imageViews = ["imageView":imageView]

self.view.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:[imageView(32)]-6-|", options: [], metrics: nil, views: imageViews))
self.view.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:[imageView(32)]-6-|", options: [], metrics: nil, views: imageViews))
```

And here is the result (the red box in the bottom right corner):

![simulator screen shot 21 02 2017 20 10 27](https://cloud.githubusercontent.com/assets/392542/23180680/744e3b68-f872-11e6-9b04-de8b73179948.png)

**Comment by marbetschar** [comment]
Any idea why it's not working as expected? Or am I doing something wrong?

**Comment by marbetschar** [comment]
just noticed it seems to work if I set the image after initialization of the imageView as follows:

```
imageView.image = UIImage(named: "my-image")
```

Seems like a bug.

**Comment by marbetschar** [comment]
For some reason this library does not compile for an archived app version when checked out via `Carthage`. FWIW: I've subclassed your as follows - this refuses to compile for the app store:

```
import UIKit
import UIImageViewAlignedSwift

class ChartImageView: UIImageViewAligned{
	
	var imageMaskingColorComponents: [CGFloat]?
	var imageTintColor: UIColor?
	
	override var image: UIImage?{
		get{
			return super.image
		}
		set{
			var newImage = newValue
			
			if let color = self.imageTintColor{
				newImage = newImage?.applied(tintColor: color)
			} else if let colorComponents = self.imageMaskingColorComponents{
				newImage = newImage?.masked(withColorComponents: colorComponents)
			}
			
			super.image = newImage
		}
	}
}
```

**Comment by piotrros** [comment]
I've just stepped into the same problem. Thanks for the hint. Seems like a bug.

**Comment by sochalewski** [comment]
@marbetschar @piotrros It's fixed since 0.4.0. In addition `scaling` is not available, as long as it doesn't work as desired.
Not sure about Carthage, because I don't use it and my knowledge about this is really poor. Please PR if you know how to fix this 👍
